src/context.tsx

```diff
@@ -59,6 +59,9 @@ const initialState: State = {
     metaAnalyzedRecords: [],
     isLoading: false
   },
+  // TODO: replace this with an obj
```

Comment:
Has an Airtable ticket been created for this?

Reply:
Will be addressed by https://airtable.com/tbli2lWQHAqBa6ZcI/viw7Tba8QVTJ49F4v/recUuEvWgjzm3dLhN